src/styles/scss/Form.scss

```diff
@@ -7,6 +7,39 @@ $primary-background: #fff;
 
 form { margin: 25px 0 35px; }
 
+.loader {
+  margin-top: 0.2em;
+  margin-left: 0.2em;
+  vertical-align: bottom;
+  display: inline-block;
+  width: 33px;
+  height: 33px;
+
+  img {
+    width: 33px;
+    height: 33px;
+  }
+}
+
+.loading {
```

Comment:
@verbman Where abouts is `.loading` being used?

Reply:
See line 53 of App.js